gatsby-node.js

```diff
@@ -326,6 +326,10 @@ exports.createSchemaCustomization = ({ actions }) => {
     translationType: String
     dataSource: String
     isTutorial: Boolean
+    features: Array
```

Comment:
Looks like the build is failing for these types, i think we have to get more explicit and do something like `[String]` like on line 315

Reply:
assuming these are an array of strings since its frontmatter

Reply:
Thank you! Update made! 🤞